src/Raven.Studio/wwwroot/App/views/partial/backupDestinations.html

```diff
@@ -118,7 +118,14 @@ <h3>Destination</h3>
                         <div data-bind="template: { name: 'backup-via-configuration-script' }"></div>
                     </div>
                     <div data-bind="if: !hasConfigurationScript()">
-                        
+                        <div class="row margin-bottom" data-bind="collapse: useCustomS3Host">
+                            <div class="col-xs-offset-12 col-sm-offset-6 col-lg-offset-4">
+                                <div class="toggle margin-left">
+                                    <input id="forcePathStyle" type="checkbox" data-bind="checked: forcePathStyle">
```

Comment:
can we include some tooltip what's that, do I need this etc?

Reply:
![image](https://user-images.githubusercontent.com/1250431/143386775-6b5bc55f-2beb-4b05-9a00-10bb0a34e495.png)

Reply:
done